src/styles/blocks/menu.scss

```
@@ -0,0 +1,37 @@
+.menu {
+  box-sizing: border-box;
+  height: 100vh;
+  background-color: #c2c2c2;
+  font-weight: 600;
+  text-transform: uppercase;
+  transition: opacity 300ms linear;
+}
+
+.menu__nav {
+  margin-bottom: 40px;
+}
+
+.menu__content {
+  padding-inline: 20px;
+  color: #1d1d1d;
+}
+
+.menu__phone-number {
+  color: inherit;
+  text-decoration: none;
+  display: block;
+  font-size: 16px;
+  line-height: 27px;
+  margin-bottom: 16px;
+}
+
+.menu__book-test {
```

You need to use nesting to avoid duplicating selector names.
Need to fix it everywhere
```suggestion
.menu {
  box-sizing: border-box;
  height: 100vh;
  background-color: #c2c2c2;
  font-weight: 600;
  text-transform: uppercase;
  transition: opacity 300ms linear;
  
  &__nav {
    .......
  }
  
  &__content {
    ........
  }
  
  &__phone-number {
    .........
  }
  
  &__book-test {
    .......
  }
}
```